ui_framework/.babelrc

```diff
@@ -1,3 +0,0 @@
-{
-  "presets": ["react", "@spalger/babel-presets"]
-}
```

Comment:
Weird, doesn't fail for me. It shouldn't have it's own babel config, but rely on Kibana's config (as that's what the code will actually end up using when run outside the tests).

I'll see if I can reproduce. Did you `npm install` before testing?